packages/go_router/tool/run_tests.dart

```diff
@@ -9,25 +9,103 @@
 // ignore_for_file: avoid_print
 
 import 'dart:io';
+import 'package:io/io.dart' as io;
 import 'package:path/path.dart' as p;
+import 'package:yaml_edit/yaml_edit.dart';
 
 Future<void> main(List<String> args) async {
-  if (!Platform.isMacOS) {
-    print('This test can only be run on macOS.');
+  if (!Platform.isMacOS && !Platform.isWindows) {
+    print('This test can only be run on macOS or windows.');
     exit(0);
   }
+  final p.Context ctx = p.context;
```

Comment:
If you are always using the default context, what's the purpose of doing everything via `Context` at all, vs using the top-level methods? Usually this pattern is for forcing a specific context that doesn't match the host platform, which you aren't doing.